modules/ROOT/pages/sap/sap-s4hana-soap-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,72 @@
+= Upgrade SAP S/4HANA SOAP Cloud Connector to 2.0 - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Upgrade Anypoint Connector for SAP S/4HANA SOAP Cloud to take advantage of the increased security offered by using OAuth 2.0 authentication to connect to SAP S/4HANA SOAP. 
+
+== Supported Upgrade Paths
+
+This is a direct upgrade from version 1.0.0 version to 2.0.0. 
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|1.0.0 |2.0.0
+|===
+
+== Changes in This Release
+
+* OAuth 2.0 is supported. +
+The *OAuth 2.0 Application Connection Provider* option has been added to *Connection* types. For information about the configuration parameters for this connection type, see xref:sap/sap-s4hana-soap-connector-reference.adoc[SAP S/4HANA SOAP Cloud Connector Reference]. 
+* Mule runtime engine (Mule) 4.2.1 or later is required. 
+* The base package name has been changed to `com.mulesoft.connector.sap.s4hana.soap`, so now all exported classes now have different fully qualified names. 
```

Comment:
```suggestion
* The base package name has been changed to `com.mulesoft.connector.sap.s4hana.soap` so that all exported classes now have different fully qualified names. 
```